packages/calcite-components/src/components/carousel/carousel.stories.ts

```diff
@@ -2,24 +2,25 @@ import { boolean, modesDarkDefault } from "../../../.storybook/utils";
 import { html } from "../../../support/formatting";
 import { placeholderImage } from "../../../.storybook/placeholderImage";
 import { ATTRIBUTES } from "../../../.storybook/resources";
+import { Carousel } from "./carousel";
 const { arrowType } = ATTRIBUTES;
 
-interface CarouselArgs {
-  controlOverlay: boolean;
-  disabled: boolean;
-  autoPlayDuration: number;
-  autoPlay: boolean;
-  label: string;
-  arrowType: string;
+interface CarouselStoryArgs {
+  controlOverlay: Carousel["controlOverlay"];
+  disabled: Carousel["disabled"];
+  autoplayDuration: Carousel["autoplayDuration"];
+  autoplay: boolean;
```

Comment:
`Carousel` should have an `autoplay` prop.

This might apply to other classes/props too (e.g., `Tooltip`).